yellowbrick/contrib/missing/dispersion.py

```diff
@@ -0,0 +1,226 @@
+
+import numpy as np
+
+from yellowbrick.style.palettes import color_palette
+from .base import MissingDataVisualizer
+
+
+##########################################################################
+## MissingValues Visualizer
+##########################################################################
+
+class MissingValuesDispersion(MissingDataVisualizer):
+    """
+    The Missing Values Dispersion visualizer shows the locations of missing (nan)
+    values in the feature dataset by the order of the index.
+
+    When y targets are supplied to fit, the output dispersion plot is color
+    coded according to the target y that the element refers to.
+
+    Parameters
+    ----------
+    alpha : float, default: 0.5
+        A value for bending elments with the background.
+
+    marker : matplotlib marker, default: |
+        The marker used for each element coordinate in the plot
+
+    classes : list, default: None
+        A list of class names for the legend.
+        If classes is None and a y value is passed to fit then the classes
+        are selected from the target vector.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Attributes
+    ----------
+    features_ : np.array
+        The feature labels ranked according to their importance
+
+    classes_ : np.array
+        The class labels for each of the target values
+
+    Examples
+    --------
+
+    >>> from yellowbrick.contrib.missing import MissingValuesDispersion
+    >>> visualizer = MissingValuesDispersion()
+    >>> visualizer.fit(X, y=y)
+    >>> visualizer.poof()
+    """
+
+    def __init__(self, alpha=0.5, marker="|", classes=None, **kwargs):
+
+        super(MissingValuesDispersion, self).__init__(**kwargs)
+        self.alpha = alpha
+        self.marker = marker
+
+        self.classes_ = classes
+
+        # Convert to array if necessary to match estimator.classes_
+        if self.classes_ is not None:
+            self.classes_ = np.array(classes)
+
+        # Set up classifier score visualization properties
+        if self.classes_ is not None:
+            n_colors = len(self.classes_)
+        else:
+            n_colors = None
+
+        self.colors = color_palette(kwargs.pop('colors', None), n_colors)
+
+
+    def get_nan_locs(self, **kwargs):
+        """Gets the locations of nans in feature data and returns
+        the coordinates in the matrix
+        """
+        if np.issubdtype(self.X.dtype, np.string_) or np.issubdtype(self.X.dtype, np.unicode_):
+            mask = np.where( self.X == '' )
+            nan_matrix = np.zeros(self.X.shape)
+            nan_matrix[mask] = np.nan
+
+        else:
+            nan_matrix = self.X.astype(float)
+
+        if self.y is None:
+            return np.argwhere(np.isnan(nan_matrix))
+        else:
+            nan_locs = []
```

Comment:
I think the user would also be interested in accessing this property, e.g. `self.nan_locs_` as a learned parameter.